### ISSUE TITLE
Pfad fpür die SPLIT Daten bei zml2lido

### ISSUE THREAD

**Issue by FrankvonHagel** [mokko/zml2lido#78]
Nichts was mich , es irritiert. Aktuell ist der Ausgabepfad der SPLIT-Daten folgender:
H:\Documents\py\zml2lido\sdata\KB-Mode\20221116\FS0002\Home\MDFRAHAG\Documents\py\zml2lido\sdata\KB-Mode\20221116\split

**Comment by mokko** [comment]
Ja. Das ist der bekannte Fehler. Den hatte ich mir schon notiert und auch schon damit herumgespielt. Leider ohne großen Erfolg. Als nächstes versuchen wir mal zml2lido bei dir auf c zu installieren. Aber vielleicht nicht heute.

**Comment by mokko** [comment]
Was dahintersteckt ist dass aus unbekannten Gründen Saxon auf Franks Rechner anders arbeitet als bei mir und relative Pfade anders verarbeitet. Das haben wir bei der Erzeugung der Split-Dateien mit xsl: result-document und beim Einbinden von vocmap.XML per XML:document.

Wir benutzen beide Java Saxon, allerdings mit unterschiedlichen Unterversionen. Außerdem hat Frank auf Netzwerklaufwerk installiert, was UNC Pfad erzwingt. Ich kann das Problem bei Installation auf Netzwerk-Pfad aber trotzdem nicht nachstellen.

**Comment by mokko** [comment]
Man könnte mal .Net Variante von Saxon ausprobieren… könnte sein, dass sie mit uns Pfaden besser zurecht kommt

**Comment by mokko** [comment]
Ich weiß nicht, was hier die Lösung war. In dem Beispiel oben ist der Pfad des split-Verzeichnisses viel zu lang, also verkehrt. Aber immerhin irgendwo im richtigen Verzeichnis (sdata).
